Description: Fix building the ODK java desktop examples against OpenJDK 11
Author: Olivier Tilloy <olivier.tilloy@canonical.com>
Bug-Ubuntu: https://bugs.launchpad.net/ubuntu/+source/libreoffice/+bug/1796361

--- a/odk/examples/DevelopersGuide/OfficeDev/DesktopEnvironment/Makefile
+++ b/odk/examples/DevelopersGuide/OfficeDev/DesktopEnvironment/Makefile
@@ -66,7 +66,6 @@ APP1_JAVAFILES  = \
 APP1_CLASSFILES = $(patsubst %.java,$(OUT_APP_CLASS)/%.class,$(APP1_JAVAFILES))
 APP1_CLASSNAMES = $(patsubst %.java,%.class,$(APP1_JAVAFILES)) \
 		CustomizeView$(QUOTE)$$ClickListener.class \
-		DocumentView$(QUOTE)$$1.class \
 		DocumentView$(QUOTE)$$Reactor.class
 
 
